docs/src/visualization.md

```diff
@@ -60,6 +60,17 @@ For further details on both of these see below:
 ```julia
 julia> pd = PlotData2D(sol)
 ```
+You can also create a `PlotData2D` by object by specifying an `iter::Int`
+or `time::Real` argument, e.g.,
+```julia
+julia> pd = PlotData2D(sol, 2) # iter::Int
+julia> pd = PlotData2D(sol, 0.5) # time::Real
+```
+If `iter=2` is specified, this will create a `PlotData2D` instance from the
+2nd saved time-step. If `time=0.5` is specified, it will construct a `PlotData2D`
+instance  using OrdinaryDiffEq.jl's interpolation to evaluate the solution at
+time `t=0.5`.
```

Comment:
We should probably also document the `PlotData2D(u, semi)` interface (but this could be done in another PR).